### DISCUSSION TITLE
Possible Infinite "Connecting..." Fix

### DISCUSSION THREAD

**Discussion by Larsundso** [maximmax42/Discord-CustomRP#141] [General]
Fix 1: 
- Use another Discord Client, you have 4 clients to try with
https://discord.com/api/downloads/distributions/app/installers/latest?channel=stable&platform=win&arch=x86
https://discord.com/api/downloads/distributions/app/installers/latest?channel=ptb&platform=win&arch=x86
https://discord.com/api/downloads/distributions/app/installers/latest?channel=canary&platform=win&arch=x86
https://discord.com/api/downloads/distributions/app/installers/latest?channel=development&platform=win&arch=x86

Fix 2: 
Disclaimer: I do use multiple Discord Clients, so maybe it's not a fix if you only use one client

Open the Pipe window by Pressing CTRL+Shift and Connect
Cycle through every option from -1 to 9
on every Option wait a few seconds until it either says Connection Error, shows your Discord Tag or still says Connecting
If it shows Error or still says Connecting, check the next Pipe, 
If it shows your Discord Tag, click Ok and Connect

**Comment by maximmax42** [comment]
Usually adding the app as an antivirus exception fixes that problem.

**Reply by Larsundso** [reply]
just an idea to give more fix options
